OrbitAccessibility/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(OrbitAccessibility CXX)
+add_library(OrbitAccessibility STATIC)
+
+target_compile_options(OrbitAccessibility PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_include_directories(OrbitAccessibility PUBLIC 
+        ${CMAKE_CURRENT_SOURCE_DIR}/include)
+
+target_include_directories(OrbitAccessibility PRIVATE 
+        ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_sources(
+  OrbitAccessibility
+  PUBLIC include/OrbitAccessibility/AccessibleInterface.h
+         include/OrbitAccessibility/AccessibleInterfaceRegistry.h
+         include/OrbitAccessibility/AccessibleObjectFake.h
+         include/OrbitAccessibility/AccessibleWidgetBridge.h
+         )
+
+target_sources(
+  OrbitAccessibility
+  PRIVATE AccessibleInterface.cpp
+          AccessibleInterfaceRegistry.cpp)
+
+target_link_libraries(
+  OrbitAccessibility
+  PUBLIC OrbitBase)
+
+add_executable(OrbitAccessibilityTests)
+
+target_compile_options(OrbitAccessibilityTests PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_sources(OrbitAccessibilityTests PRIVATE
+               include/OrbitAccessibility/AccessibleObjectFake.h)
```

Comment:
Do you need this again? It's already in the sources of `OrbitAccessibility`.

Reply:
No, missed that, thanks